automata/symbol/graph/symbol_graph.py

```diff
@@ -20,6 +28,16 @@
 logger = logging.getLogger(__name__)
 
 
+def _load_index_protobuf(path: str) -> Index:
```

Comment:
we might like to make this a staticmethod. I still prefer having static methods because it groups the code nicely for LLMs.

Reply:
Happy to revert this—I only changed it to better fit Google's style guide.